app/src/main/java/com/berniesanders/connect/MainActivity.java

```diff
@@ -33,8 +35,13 @@
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        DaggerActivityComponent.builder()
```

Comment:
I guess it's partly a limitation of dagger 2. I agree we should be holding onto the component in case we need to do another injection. I'll update that tonight.